frontend/packages/component-library/package.json

```diff
@@ -205,6 +205,15 @@
       "import": "./dist/esm/formFieldWrapper/index.mjs",
       "require": "./dist/cjs/formFieldWrapper/index.js"
     },
+    "./cms/image": {
+      "types": {
+        "import": "./dist/esm/cms/image/index.d.ts",
+        "require": "./dist/cjs/cms/image/index.d.ts"
+      },
+      "import": "./dist/esm/cms/image/index.mjs",
+      "require": "./dist/cjs/cms/image/index.js"
+    },
+    "./cms/image/index.css": "./dist/esm/cms/image/index.css",
```

Comment:
nit: looks like we don't need to export `index.css` file now as it's being imported directly in `index.ts` file